<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_requirement</class>
 <widget class="QWidget" name="form_requirement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>39</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hlayout_id">
     <item>
      <widget class="QLabel" name="lbl_id">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_system">
     <item>
      <widget class="QLabel" name="lbl_system">
       <property name="text">
        <string>System:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_system"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_section">
     <item>
      <widget class="QLabel" name="lbl_section">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_section"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_definition">
     <item>
      <widget class="QLabel" name="lbl_definition">
       <property name="text">
        <string>Definition:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_definition"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_requirement">
     <property name="leftMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
